chemprop/data/utils.py

```diff
@@ -94,6 +98,9 @@ def get_task_names(path: str,
 
     target_names = [column for column in columns if column not in ignore_columns]
 
+    if loss_function == "quantile_interval":
+        target_names = target_names * 2
+
```

Comment:
Okay, so here is where you double the number of tasks named. Presumably to have a low quantile task and a high quantile task for each. I'm having trouble finding where the targets are doubled to match it though, where would that be located?

Reply:
The function of interest is get_data in data/utils.py. Targets is created by querying each task in target_names. So doubling target_names also doubles targets when it is created.

Reply:
Oh I see. Okay, this now. Thanks.